.mci.yml

```diff
@@ -672,7 +671,7 @@ functions:
                           export LD_LIBRARY_PATH="$(pwd)/build/install/lib:$LD_LIBRARY_PATH"
                       fi
                   fi
-                  # The example projects never run under valgrind, since we haven't added execution
+                  # The example projects never run under asan/ubsan, since we haven't added execution
                   # logic to handle ${test_params}.
```

Comment:
This comment also no longer seems relevant. The `${test_params}` for ASAN and UBSAN only include environment variables. I suggest adding `${test_params}` below to include the environment variables:
```
                  if [[ -z "$MONGODB_API_VERSION" ]]; then
                    echo "Building example projects..."
                    ${test_params} .evergreen/build_example_projects.sh
                    echo "Building example projects... done."
                  fi
```